modules/programs/mise.nix

```
@@ -3,16 +3,16 @@
 with lib;
 
 let
-  cfg = config.programs.rtx;
+  cfg = config.programs.mise;
   tomlFormat = pkgs.formats.toml { };
 in {
   meta.maintainers = [ hm.maintainers.pedorich-n ];
 
```

Should add a `mkRemovedOptionModule` import to notify users about the move. It may be sufficient to notify about the `rtx.enable` option. Something like

```suggestion

  imports = [
    (mkRemovedOptionModule [ "programs" "rtx" "enable" ] ''
      The rtx package has been replaced by mise, please switch over to
      using the options under

          programs.mise.*

      instead.
    '')
  ];

```

perhaps.

---

I don't know how compatible the configuration is between rtx and mise. If all rtx confiurations are valid mise configurations, then you could even use `mkRenamedOptionModule` for the different rtx options.

---

Thanks! That's exactly what I need.
Given the split of the configuration file now they're not exactly the same anymore. So I think it's better to use `mkRemovedOptionModule`